.github/workflows/test.yml

```diff
@@ -47,6 +47,7 @@ jobs:
         uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
+          bundler: '2.3'  # FIXME: Delete after release containing https://github.com/ruby/setup-ruby/pull/423
```

Comment:
Looks like this is released (https://github.com/ruby/setup-ruby/commit/8d6236c5db46376078b34bf85d247e3ab767cb95 is part of 1.129.0)